builds/jenkins-build-pack-sls-app/Jenkinsfile

```diff
@@ -632,7 +632,7 @@ def validateDeploymentConfigurations(def prop) {
 		if (_runtime == "") {
 			error "Wrong configuration. Value for Key 'providerRuntime' is missing in the configuration"
 		} else {
-			def validRuntimes = ["nodejs6.10", "nodejs8.10", "python2.7", "python3.6", "java8", "go1.x"]
+			def validRuntimes = ["nodejs8.10", "nodejs10.x", "python3.6", "java8", "go1.x"]
```

Comment:
Will this break build for services older runtimes?

Reply:
Not really. At this point, Jazz doesn't support stack upgrades completely (which means there won't be any "old"  services when the stack starts).

Reply:
But I guess the same is not true with Internal?